src/ClientAssetsManager.php

```diff
@@ -203,13 +207,13 @@ private function initJavaScriptOptimization()
             }
             $cachePath = $this->cacheDir . '/' . sha1(implode('|', $hash)) . '.js';
             if (!is_file($cachePath)) {
-                // Combined script is not yet available - generate it and put into cache
-                $content = [];
+                // Combined script is not yet available - generate, minified it and put into cache
+                $content = '';
                 foreach ($parts as $part) {
-                    $content[] = '/* [' . basename($part) . '] */;';
```

Comment:
This comment is intentional because it allows to understand where the code came from